remo/events/migrations/0024_migrate_pads_to_etherpad_lite.py

```diff
@@ -0,0 +1,205 @@
+# -*- coding: utf-8 -*-
+import datetime
+import re
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+from django.conf import settings
+
+
+PAD_RGX = re.compile('(http|https)://(.*)etherpad.mozilla.org/(.*)')
+
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        """ Migrate the planning pad in events to the new etherpad url."""
+        events = orm['events.Event'].objects.all()
+        for event in events:
+            match = PAD_RGX.match(event.planning_pad_url)
+            new_planning_pad = ''
+            if match:
+                new_planning_pad = settings.ETHERPAD_URL + match.groups()[2]
```

Comment:
You might need to check if ``match.groups()[2]`` exist in case someone added a wrong url. Else the datamigration will fail.